pkg/install/generate.go

```diff
@@ -362,6 +377,10 @@ func GenerateUninstallerJobForDeprovisionRequest(
 		completeAWSDeprovisionJob(req, job)
 	case req.Spec.Platform.Azure != nil:
 		completeAzureDeprovisionJob(req, job)
+	case req.Spec.Platform.GCP != nil:
+		completeGCPDeprovisionJob(req, job)
+	default:
+		return nil, errors.New("deprovision requests currently supported for platform")
```

Comment:
Missing a "not" I think.

Reply:
You have clear e2e so lets merge this and get that one word in another followup.